Dockerfile

```diff
@@ -13,33 +14,33 @@ ARG OPM_VERSION=v1.26.3
 
 ENV POLICY_PATH="/project"
 
-RUN ARCH=$(uname -m) && curl https://mirror.openshift.com/pub/openshift-v4/"$ARCH"/clients/ocp/stable/openshift-client-linux.tar.gz --output oc.tar.gz && tar -xzvf oc.tar.gz -C /usr/bin && rm oc.tar.gz && \
-    curl -LO "https://github.com/bats-core/bats-core/archive/refs/tags/v$BATS_VERSION.tar.gz" && \
-    curl -L https://github.com/operator-framework/operator-registry/releases/download/"${OPM_VERSION}"/linux-amd64-opm > /usr/bin/opm && chmod +x /usr/bin/opm && \ 
-    curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" && \
-    tar -xf "v$BATS_VERSION.tar.gz" && \
-    cd "bats-core-$BATS_VERSION" && \
-    ./install.sh /usr && \
-    cd .. && rm -rf "bats-core-$BATS_VERSION" && rm -rf "v$BATS_VERSION.tar.gz" && \
-    dnf -y --setopt=tsflags=nodocs install \
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    microdnf -y --setopt=tsflags=nodocs install \
```

Comment:
`--setopt=install_weak_deps=False`  Maybe this opt can reduce size more

Reply:
669 > 648